authors/apps/authentication/tests/test_models.py

```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase
+
+from ..models import User
+
+class UserModelTest(TestCase):
+
+    
+    def setUp(self):
+        """ Creates an instance of the User """
+
+        self.user = User(username="test_user", email="test.sims@andela.com", password="testman101")
+        self.user.save()
+    
+
+    def test_user_creation(self):
+        """test that user is created"""
+
+        self.assertTrue(self.user, User)
```

Comment:
I think this test is redundant because it is not testing any output but rather testing what you've already done in the `setUp`.

Reply:
ok let look into that